src/Components/PaperEditor/TranscriptsContainer/TranscriptTabContent/TranscriptMenu.js

```diff
@@ -12,23 +12,9 @@ import { faHighlighter, faCog } from '@fortawesome/free-solid-svg-icons';
 
 const TranscriptMenu = (props) => {
   const labels = props.labels;
+  // needs refactor to do if/else logic of rendering LabelsList here
```

Comment:
Can we delete this note?

Reply:
we need to keep it until the below state is refactored - 
`const [ isLabelsListOpen, setIsLabelsListOpen ] = useState(true);` which is only used to decide to render LabelsList :)